docs/CHANGELOG.asciidoc

```diff
@@ -8,53 +8,606 @@
+=== Deprecations
+
+The following functionality is deprecated in 7.14.0, and will be removed in 8.0.0.
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend
+you make the necessary updates after you upgrade to 7.14.0.
+
+[discrete]
+[[deprecation-103740]]
+.Deprecates `map.proxyElasticMapsServiceInMaps`
+[%collapsible]
+====
+*Details* +
+The `map.proxyElasticMapsServiceInMaps` setting is deprecated. For more information, refer to {kibana-pull}103740[#103740].
+
+*Impact* +
+Remove the `map.proxyElasticMapsServiceInMaps` setting in kibana.yml, the CLI flag, or the Docker environment variable, the host Elastic Maps Service locally.
+====
+      
+[discrete]
+[[deprecation-103721]]
+.Deprecates `apm_oss.fleetMode`
+[%collapsible]
+====
+*Details* +
+The `apm_oss.fleetMode` setting in the APM OSS plugin is deprecated. For more information, refer to {kibana-pull}103721[#103721].
+
+*Impact* +
+Remove the `apm_oss.fleetMode` setting from the APM OSS plugin.
+====
+      
+[discrete]
+[[deprecation-103373]]
+.Deprecates the 'map.regionmap' setting and Configured GeoJSON layer
+[%collapsible]
+====
+*Details* +
+The 'map.regionmap' setting and Configured GeoJSON layer are deprecated. For more information, refer to {kibana-pull}103373[#103373].
+
+*Impact* +
+
+. Remove the 'map.regionmap' setting in kibana.yml, the CLI flag, or Docker environment variable.
+. To upload each layer defined by `map.regionmap.layers`, use Upload GeoJSON.
+. To update all maps that use Configured GeoJSON, use the Choropleth layer wizard to build a replacement layer, then delete the Configured GeoJSON layer from your map.
+====
+      
+[discrete]
+[[deprecation-102878]]
+.Deprecates Ingest Pipelines URL generator
+[%collapsible]
+====
+*Details* +
+The Ingest Pipelines URL generator is deprecated. For more information, refer to {kibana-pull}102878[#102878].
+
+*Impact* +
+The Machine Learning link that navigates you to *Ingest Managment* now uses locators.
+====
+      
+[discrete]
+[[deprecation-102854]]
+.Deprecates Dashboard URL generator
+[%collapsible]
+====
+*Details* +
+The Dashboard URL generator is deprecated. For more information, refer to {kibana-pull}102854[#102854].
+
+*Impact* +
+Drilldowns now use the *Dashboard* locator.
+====
+      
+[discrete]
+[[deprecation-102712]]
+.Deprecates Discover URL generator
+[%collapsible]
+====
+*Details* +
+The *Explore Underlying Data* URL generator is deprecated. For more information, refer to {kibana-pull}102712[#102712].
+
+*Impact* +
+The *Explore Underlying Data* interaction now uses the *Discover* locator.
+====
+      
+[discrete]
+[[deprecation-102059]]
+.Deprecates `csp.rules`
+[%collapsible]
+====
+*Details* +
+The `csp.rules` configuration property is deprecated. You are no longer required to set the default {kib} values. For more information, refer to {kibana-pull}102059[#102059].
+
+*Impact* +
+Use `csp.script_src`, `csp.worker_src`, and `csp.style_src`.
```

Comment:
Should we add a link to where these are defined in `docs/setup/settings.asciidoc`?

Reply:
Yes! I'll add this as a best practice to our release notes process.